File: 프로그래머스/0/120892. 암호 해독/암호 해독.java
class Solution {
    public String solution(String cipher, int code) {
        String answer = "";
        
        int num = 0;
    
        for (int i = 1; i <= cipher.length(); i++) {
            num = code * i - 1;
            if (num >= cipher.length()) {
                break;
            }
            answer += cipher.charAt(num);
            // answer += cipher[num].toString;
    }
    
        
        return answer;
    }
}

File: 프로그래머스/0/120892. 암호 해독/암호 해독.js
function solution(cipher, code) {
    var answer = '';
    var num = 0;
    
    for (i = 1; i <= cipher.length; i++) {
        num = code * i - 1;
        if (num >= cipher.length) {
            break;
        }
        answer += cipher[num];
        // answer += cipher[num].toString;
    }
    
    
    return answer;
}